setup.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 from setuptools import Command, find_packages, setup
 
-version = '0.8.0'
+version = '0.8.0-bf3'
```

Comment:
Please drop this edit (I assume it was for local testing).

I'll be prepping an 0.8.1 release this week to get your change released.